Source/Fuse.Nodes/Visual.Children.uno

```diff
@@ -258,18 +250,16 @@ namespace Fuse
 					var c = items.Current;
 					if (allowMove)
 					{
-						var where = IndexOf(c);
-						if (where != -1)
+						if (Children_Contains(c))
```

Comment:
That would be a separate optimziation for later. I don't want the extra code churn and added field on this branch, and the HashSet doesn't show up as significant in any profiling.